app/build.gradle

```diff
@@ -75,6 +75,11 @@ android {
     buildFeatures {
         dataBinding true
         viewBinding true
+        compose true
+    }
+
+    composeOptions {
+        kotlinCompilerExtensionVersion rootProject.ext.composeUiVersion
```

Comment:
```suggestion
        kotlinCompilerExtensionVersion rootProject.ext.composeUiVersion
        
        lintOptions {
            disable "DaggerKotlinIssueDetector"
        }
```

I looked into the linting error some more and this seemed to help!

Reply:
Nice, thanks for looking into the lint issue!